specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4573,14 +4573,38 @@
           "type": "string",
           "description": "The namespace of the resource provider."
         },
+        "authorizations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceProviderAuthorization"
+          },
+          "description": "The authorizations fields of the resource provider."
+        },
+        "metadata": {
+          "type": "object",
+          "description": "The metadata."
+        },
         "registrationState": {
           "readOnly": true,
           "type": "string",
+          "enum": [
```

Comment:
Adding enum values will be breaking change.

Reply:
The enum values were missing earlier. I have to add them.